comp/core/workloadmeta/collectors/internal/process/stub.go

```diff
@@ -0,0 +1,7 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Why do we need this file ?

Reply:
Franky, I'm not totally sure. I did this because every other collector does it as well. I think its to ensure the package exists in every build? Would you happen to know @adel121 ?